samples/shared/Mixer.h

```diff
@@ -45,15 +45,15 @@ class Mixer : public IRenderableAudio {
         }
     }
 
```

Comment:
Add comment that ownership must be managed by the caller. Do not delete the render while in use.